packages/components/src/dimension-control/README.md

```diff
@@ -0,0 +1,128 @@
+DimensionControl
+=============================
+
+`DimensionControl` is a component designed to provide a UI to control spacing and/or dimensions. It is intended for use within the editor `InspectorControls` sidebar.
```

Comment:
I think we should not refer "It is intended for use within the editor `InspectorControls` sidebar" nothing in the component makes it less usable elsewhere e.g: in settings modal.

Reply:
Agreed.